src/Nethermind/Nethermind.HealthChecks/HealthChecksWebhookInfo.cs

```diff
@@ -0,0 +1,42 @@
+/* Class to provide useful information in healh checks' webhook notifications */
+
+using System.Net;
+using System;
+
+namespace Nethermind.HealthChecks
+{
+    public class HealthChecksWebhookInfo
+    {
+
+        private string _description;
+        private string _ip;
+        private string _hostname;
+        private string _nodeName;
+
+        public HealthChecksWebhookInfo(string description, IPAddress externalIp, string nodeName)
```

Comment:
I suggest using _api.IpResolver and IMetricsConfig in constructor instead of IPAddress and nodeName